dbt/models/default/columns.md

```diff
@@ -0,0 +1,384 @@
+- `235.4 - OTHER-MINOR NEW CONSTRUCTION - signs`
+- `235.5 - OTHER-MINOR NEW CONSTRUCTION - canopies`
+- `235.6 - OTHER-MINOR NEW CONSTRUCTION - cell towers (monopoles)`
+- `235.7 - OTHER-MINOR NEW CONSTRUCTION - sprinkler system`
+- `235.8 - OTHER-MINOR NEW CONSTRUCTION - security system`
+- `235.9 - OTHER-MINOR NEW CONSTRUCTION - handicap ramps`
+- `251 - PARTITIONING`
+- `252 - DECONV/CONV AMT. LIV. UT.`
+- `253 - AIR CONDITIONING - FIREPLACES`
+- `253.1 - AIR CONDITIONING - FIREPLACES - fireplaces`
+- `253.2 - AIR CONDITIONING - FIREPLACES - air conditioning`
+- `254 - BASEMENT-ATTIC ROOM BATH`
+- `254.1 - BASEMENT-ATTIC ROOM BATH - basement rooms`
+- `254.2 - BASEMENT-ATTIC ROOM BATH - bathrooms`
+- `254.3 - BASEMENT-ATTIC ROOM BATH - attic rooms`
+- `255 - REPLACE-REPAIR BUIDLING - FACTORY`
+- `255.1 - REPLACE-REPAIR BUIDLING - FACTORY - rehab`
+- `255.2 - REPLACE-REPAIR BUIDLING - FACTORY - replacement`
+- `255.3 - REPLACE-REPAIR BUIDLING - FACTORY - repair to existing building`
+- `255.4 - REPLACE-REPAIR BUIDLING - FACTORY - replacement minor`
+- `255.5 - REPLACE-REPAIR BUIDLING - FACTORY - repair to existing building major`
+- `255.6 - REPLACE-REPAIR BUIDLING - FACTORY - repair to existing building minor`
+- `256 - OTHER REMODELING`
+- `256.1 - OTHER REMODELING - elevators`
+- `256.2 - OTHER REMODELING - escalators`
+- `256.3 - OTHER REMODELING - handicap ramps`
+- `256.4 - OTHER REMODELING - awnings`
+- `256.5 - OTHER REMODELING - fire alarms`
+- `256.6 - OTHER REMODELING - security alarms`
+- `256.7 - OTHER REMODELING - fire alarms - Hood Suppression System`
+- `256.8 - OTHER REMODELING - fire alarms - Sprinkler for Fire`
+- `256.9 - OTHER REMODELING - fire alarms - Security alarms`
+- `271 - MAJOR IMPROVEMENT - WRECK`
+- `272 - MINOR IMPROVEMENT - WRECK`
+- `291 - MAJOR IMPROVEMENT - BURNOUT`
+- `292 - MINOR IMPROVEMENT - BURNOUT`
+- `311 - CARRIER - NEW MAJOR CONSTRUCTION`
+- `312 - NON-CARRIER NEW MAJOR CONSTRUCTION`
+- `331 - CARRIER - REMODELING`
+- `332 - NON-CARRIER-REMODELING`
+- `351 - CARRIER - REMODELING`
+- `352 - NON-CARRIER - REMODELING`
+- `371 - MAJOR IMPROVEMENT WRECK`
+- `372 - MINOR IMPROVEMENT - WRECK`
+- `391 - MAJOR IMPROVEMENT - BURNOUT`
+- `392 - MINOR IMPROVEMENT - BURNOUT`
+- `400 - EXEMPT PROPERTIES`
+- `411 - LEASEHOLD`
+- `500 - OFFICE PERMITS`
+- `510 - RECHECKS`
+- `511 - OCCUPANCY`
+- `512 - PRIOR YEAR PARTIAL ASSESSMENT`
+- `513 - NEW CONSTRUCTION`
+- `514 - ADDED IMPROVEMENTS`
+- `515 - MANUALLY ISSUED PERMIT`
+- `516 - PRIOR YEAR RECHECK`
+- `517 - DIVISION PERMIT`
+- `517.1 -DIVISION PERMIT - Petition`
+- `517.11 -DIVISION PERMIT - Petition Leasehold`
+- `517.2 -DIVISION PERMIT - Condominimum`
+- `517.3 -DIVISION PERMIT - Subdivision`
+- `517.4 -DIVISION PERMIT - Condo Removal`
+- `517.5 -DIVISION PERMIT - Petition Road Taking`
+- `517.6 -DIVISION PERMIT - Vacation`
+- `517.7-DIVISION PERMIT - Dedication`
+- `517.8 -DIVISION PERMIT - Other`
+- `517.9 -DIVISION PERMIT - Mix Class`
```

Comment:
```suggestion
- `517.1 - DIVISION PERMIT - Petition`
- `517.11 - DIVISION PERMIT - Petition Leasehold`
- `517.2 - DIVISION PERMIT - Condominimum`
- `517.3 - DIVISION PERMIT - Subdivision`
- `517.4 - DIVISION PERMIT - Condo Removal`
- `517.5 - DIVISION PERMIT - Petition Road Taking`
- `517.6 - DIVISION PERMIT - Vacation`
- `517.7 - DIVISION PERMIT - Dedication`
- `517.8 - DIVISION PERMIT - Other`
- `517.9 - DIVISION PERMIT - Mix Class`
```

Reply:
Good catch! As above, the missing spaces here match the precise formatting of the values in iasWorld; however, since the typo in this case is in the separator and not in the value itself (only the number gets preserved in our data) I think it makes sense to fix it on our side.